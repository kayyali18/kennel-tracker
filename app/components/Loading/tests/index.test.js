import React from 'react'
import { shallow } from 'enzyme'

import LoadingDots from '../index'

it('should match snapshot', () => {
  const renderedComponent = shallow(<LoadingDots />)
  expect(renderedComponent).toMatchSnapshot()
})
