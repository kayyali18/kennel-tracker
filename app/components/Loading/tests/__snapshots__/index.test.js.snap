// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<Wrapper__DotWrapper>
  <Dot
    delay="0s"
  />
  <Dot
    delay=".1s"
  />
  <Dot
    delay=".2s"
  />
  <Dot
    delay=".3s"
  />
  <Dot
    delay=".4s"
  />
</Wrapper__DotWrapper>
`;
